docs/root/version_history/v1.13.8.rst

```diff
@@ -0,0 +1,8 @@
+1.13.8 (January 15, 2021)
+=========================
+
+Changes
+-------
```

Comment:
should there be a space after Changes as in these release notes? 
https://github.com/envoyproxy/envoy/pull/14306/files

Reply:
Thanks for catching it!